MdeModulePkg/Core/PiSmmCore/PiSmmIpl.c

```diff
@@ -1437,6 +1437,7 @@ GetFullSmramRanges (
   EFI_SMM_RESERVED_SMRAM_REGION   *SmramReservedRanges;
   UINTN                           MaxCount;
   BOOLEAN                         Rescan;
+  BOOLEAN                         Failed = FALSE;

```

Comment:
In this case, please follow the pattern to separate variable declaration and initialization.

Reply:
Updated.